crates/ruff_linter/resources/test/fixtures/pep8_naming/N811.py

```diff
@@ -1,3 +1,9 @@
+# Error
+import mod.CON as c
```

Comment:
nit: it would be much easier to review your changes if you only made changes to the bottom of the fixure file. Adding lines at the top means that all lines in the snapshot change, which makes assessing the impact of your change here quite hard. Could you move this import down below the `from mod import ANOTHER_CONSTANT as another_constant` line?